api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -228,3 +254,54 @@ message Pass {
   // this pass.
   double uplink_center_frequency_hz = 10;
 }
+
+// A plan that specifies antenna configuration for tracking a satellite. A plan
+// is most often created from a pass but does not have to. A plan can contain
+// arbitrary antenna angles and communication frequencies.
+message Plan {
```

Comment:
no need to include ground station information?

Reply:
It wasn't in the spec, but it may be important. @hoshir can you confirm?

Reply:
Correct, without the information, users won't be able to see GS information of the plan. I added it to the spec.